### PR TITLE
22 add r0 r1 calibration to reader

### PR DESCRIPTION
**Pull request by tibaldo**
See #22 for a description.

### CODE REVIEW

**Comment by tibaldo** [comment]
Did anybody have a chance to look at this PR and see if any modifications are needed before it can be merged?

**Comment by jlenain** [comment]
> Did anybody have a chance to look at this PR and see if any modifications are needed before it can be merged?

Hi @tibaldo !
Sorry for the delay with this PR, and many thanks for this!
It all looks good to me, I just have one open question on how to handle "broken" pixels downstream within `ctapipe`. Maybe you have a better insight on this than me, or @kosack might enlighten me.

**Review by jlenain — APPROVED** [review]
Approved, just one open question on the handling of "broken" pixels downstream within `ctapipe`.

**Review by jlenain — APPROVED** [review]

File: ctapipe_io_nectarcam/calibration.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+                        selected_gain_channel=r1.selected_gain_channel
+                    )
+
+            # do not use pixels that are broken in this run or have unusable calibration
+            unusable_pixels = np.logical_and(
+                event.mon.tel[tel_id].pixel_status.hardware_failing_pixels,
+                self.mon_data.tel[tel_id].calibration.unusable_pixels
+            )
+            # set r1 waveforms to 0 for broken pixels
+            if r1.selected_gain_channel is None:
```
[the remark this answers is not in the document]
Comment (tibaldo) [reply ctapipe_io_nectarcam/calibration.py:129]: Thank you @jlenain. No, I admit that I'm not particularly knowledgeable about handling of broken pixels in ctapipe. Let's wait for feedback from @kosack on this.

**Comment by tibaldo** [comment]
Hi @jlenain, @kosack does not seem to be available. What do you think about merging this so that other people can test, and in parallel we can start developing/testing the software to generate the calibration files in nectarchain? I will make another pull request later to set the treatment of broken pixels according to the usage in ctapipe.

**Pull request merged**